client/my-sites/site-settings/jetpack-credentials/credentials-configured/index.jsx

```diff
@@ -89,6 +116,14 @@ class CredentialsConfigured extends Component {
 			);
 		}
 
+		const headerText = this.supportsRealtimeBackup()
+			? translate(
+					'Your site is being backed up in real time and regularly scanned for security threats.'
+			  )
+			: translate(
+					'Your site is being backed up every day and regularly scanned for security threats.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **27** times:
`translate( 'Your site is being backed up in real time and regularly scanned for security threats.' )` *ES Score: 8*

Reply:
This is expected, we're introducing a new variation of that string.